apitest/src/main/java/bisq/apitest/Scaffold.java

```diff
@@ -0,0 +1,445 @@
+            if (copyBobDataDir.run().getExitStatus() != 0)
+                throw new IllegalStateException("Could not install bob data dir");
+
+            log.info("Installed dao-setup files into {}", buildDataDir);
+
+            // Write a bitcoin.conf file with the correct path to the blocknotify script,
+            // and save it to the build resource dir.
+            installBitcoinConf();
+
+            // Copy the blocknotify script from the src resources dir to the
+            // build resources dir.  Users may want to edit it sometimes,
+            // when all default block notifcation ports are being used.
+            installBitcoinBlocknotify();
+
+        } catch (IOException | InterruptedException ex) {
+            throw new IllegalStateException("Could not install dao-setup files from " + srcResourcesDir, ex);
+        }
+    }
+
+    private void cleanDaoSetupDirectories() {
+        String buildDataDir = config.rootAppDataDir.getAbsolutePath();
+        log.info("Cleaning dao-setup data in {}", buildDataDir);
+
+        try {
+            BashCommand rmBobDataDir = new BashCommand("rm -rf " + config.rootAppDataDir + "/" + bobdaemon.appName);
+            if (rmBobDataDir.run().getExitStatus() != 0)
+                throw new IllegalStateException("Could not delete bob data dir");
+
+            BashCommand rmAliceDataDir = new BashCommand("rm -rf " + config.rootAppDataDir + "/" + alicedaemon.appName);
+            if (rmAliceDataDir.run().getExitStatus() != 0)
+                throw new IllegalStateException("Could not delete alice data dir");
+
+            BashCommand rmArbNodeDataDir = new BashCommand("rm -rf " + config.rootAppDataDir + "/" + arbdaemon.appName);
+            if (rmArbNodeDataDir.run().getExitStatus() != 0)
+                throw new IllegalStateException("Could not delete arbitrator data dir");
+
+            BashCommand rmSeedNodeDataDir = new BashCommand("rm -rf " + config.rootAppDataDir + "/" + seednode.appName);
+            if (rmSeedNodeDataDir.run().getExitStatus() != 0)
+                throw new IllegalStateException("Could not delete seednode data dir");
+
+            BashCommand rmBitcoinRegtestDir = new BashCommand("rm -rf " + config.bitcoinDatadir + "/regtest");
+            if (rmBitcoinRegtestDir.run().getExitStatus() != 0)
+                throw new IllegalStateException("Could not clean bitcoind regtest dir");
+
+        } catch (IOException | InterruptedException ex) {
+            throw new IllegalStateException("Could not clean dao-setup files from " + buildDataDir, ex);
+        }
+    }
+
+    private void installBitcoinConf() {
+        // We write out and install a bitcoin.conf file for regtest/dao mode because
+        // the path to the blocknotify script is not known until runtime.
+        String bitcoinConf = "\n"
+                + "regtest=1\n"
+                + "[regtest]\n"
+                + "peerbloomfilters=1\n"
+                + "rpcport=18443\n"
```

Comment:
rpcport is configured in ApiTestConfig, but here it's hardcoded.

Reply:
Oops.  Fixed it and two other hardcoded configs in commit cf3b545.